core-bundle/src/Resources/contao/modules/ModuleLogout.php

```diff
@@ -52,19 +52,14 @@ public function generate()
 			return $objTemplate->parse();
 		}
 
-		// Set last page visited
-		if ($this->redirectBack)
-		{
-			$_SESSION['LAST_PAGE_VISITED'] = $this->getReferer();
-		}
-
 		$strLogoutUrl = System::getContainer()->get('security.logout_url_generator')->getLogoutUrl();
 		$strRedirect = Environment::get('base');
+		$strReferer = $this->getReferer();
 
 		// Redirect to last page visited
-		if ($this->redirectBack && !empty($_SESSION['LAST_PAGE_VISITED']))
+		if ($this->redirectBack && !empty($strReferer))
```

Comment:
```suggestion
		if ($this->redirectBack && $strReferer)
```

Reply:
Changed in 0c914d124acf5c0ec87df3721b74e1cdfeebd4d7